src/Immunisatie-0-1/Cert/XIS-Server/imm-serve-5-1.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="server">
+    <id value="imm-serve-5-1"/>
+    <version value="r4-1.0.0"/>
+    <name value="Immunization - XIS Server - Scenario 2.1 - Serve Immunization resources of 'van der Voorden-der Teeuwen'"/>
+    <description value="Scenario 2.1 - Serve Immunization resources of 'van der Voorden-der Teeuwen'"/>
+    <nts:patientTokenFixture href="imm-Patient-05-token.xml"/>
+
+    <nts:include value="variable.immunization.medication.url" name="pharmaceuticalProduct-url-1" sequenceNumber="0"/>
+    <nts:include value="variable.immunization.medication.url" name="pharmaceuticalProduct-url-2" sequenceNumber="1"/>
+    <nts:include value="variable.immunization.medication.url" name="pharmaceuticalProduct-url-3" sequenceNumber="2"/>
+    <nts:include value="variable.immunization.medication.url" name="pharmaceuticalProduct-url-4" sequenceNumber="3"/>
```

Comment:
De opmerkingen bij het vorige script gelden (uiteraard) ook hier en alle volgende keren

Reply:
is verplaatst